ScrumBoardSimulator/src/main/java/se/bettercode/scrum/RandomStoryTitleGenerator.java

```diff
@@ -1,27 +1,65 @@
 package se.bettercode.scrum;
 
+import org.mockito.internal.matchers.Null;
+
+import java.io.*;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Random;
 
+import static se.bettercode.scrum.ScrumGameApplication.filePath;
+
 public class RandomStoryTitleGenerator {
 
-    static String[] verbs = {"Create", "Update", "Remove", "Refactor", "Implement", "Rewrite", "Make", "Redesign", "Reduce"};
-    static String[] subject = {"GUI", "Merchant Service", "Sendout", "Customer Callback", "Retention Service", "Checkout",
-                        "German Store", "French Store", "Nordic Suffering", "User Tracking", "Out-out Service",
-                        "Self Service", "Facebook Integration", "G+ Integration", "PSP Integration", "Bank Integration",
-                        "User ID Integration", "Deposit Service", "Statistics Overview", "Spike", "Random Idea"};
+    static String[] verbs;
+    static String[] articles;
+    static String[] subject;
+
+    public RandomStoryTitleGenerator() {
+
+        if(filePath == null) {
+            filePath = "C:\\Users\\bkanumur\\Documents\\GitHub\\SBS-Consiglio-Project1\\ScrumBoardSimulator\\src\\main\\java\\se\\bettercode\\scrum\\static\\words.txt";
```

Comment:
Can you put the txt file here: String folderPath = System.getProperty("user.home") + "/Desktop/SBS Program/words.txt?